src/server/pkg/serviceenv/config.go

```diff
@@ -68,7 +69,14 @@ type PachdSpecificConfiguration struct {
 	RequireCriticalServersOnly bool   `env:"REQUIRE_CRITICAL_SERVERS_ONLY",default=false"`
 	MetricsEndpoint            string `env:"METRICS_ENDPOINT",default="`
 	// TODO: Merge this with the worker specific pod name (PPS_POD_NAME) into a global configuration pod name.
-	PachdPodName string `env:"PACHD_POD_NAME,required"`
+	PachdPodName        string `env:"PACHD_POD_NAME,required"`
+	PostgresServiceHost string `env:"POSTGRES_SERVICE_HOST,default=172.17.0.1"`
```

Comment:
I'm surprised that there's a sensible default for this, but I'm not sure I understand this part totally. This seems like the IP for a kubernetes service called "postgres"—can you control service IPs? Or is this like different entirely?

Reply:
Removed this default, it was a leftover from before we had a better story for deploying postgres.